ocsci/pytest_customization/ocsci_logging.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os
+import time
+import pytest
+
+from ocsci import config as ocsci_config
+
+FORMATTER = logging.Formatter(
+    "%(asctime)s - %(levelname)s - %(name)s.%(funcName)s.%(lineno)d"
+    " - %(message)s"
+)
+ep_time = int(time.time())
+log_dir = "/logs_" + str(ep_time)
```

Comment:
Use fstring instead of concat and casting. `log_dir = f"/logs_{ep_time}"`

Reply:
Done